tests/end_to_end/test_vulnerability_detector/conftest.py

```diff
@@ -129,7 +247,7 @@ def host_manager(request):
     """
     inventory_path = request.config.getoption('--inventory-path')
     host_manager = HostManager(inventory_path)
-    validate_environment(host_manager)
+    # validate_environment(host_manager)
```

Comment:
Why is this line commented?

Reply:
Done 88bd94461aeb9767648ae1d537592de74f45b247